wireless_modules/middle_module/test.py

```diff
@@ -25,9 +26,18 @@ def display_difference(old, new):
 
 
 def on_msg_receive(client, data, message):
+    """
+    You can find out more details about the parameters of this function at https://pypi.org/project/paho-mqtt/#callbacks
+    """
     # Extract data from mqtt topic
     data = message.payload.decode('utf-8')
     data_dict = json.loads(data)
+
+    if message.topic == SUB_BATTERY_TOPIC:
```

Comment:
A neater solution would be to use
```python
from paho.mqtt import topic_matches_sub
if topic_matches_sub("/v3/wireless_module/+/battery", message.topic):
```
Then we wouldn't need the mutating global variable.